Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:8.16-alpine
+FROM node:10.18-alpine
```

Comment:
Why not go for latest LTS? 12.14.1 I guess

Reply:
Here I tried to do the smallest version bump possible, to see how it goes
We absolutely should go with `lts-alpine3.9` for parity with `lts/*` in .travis.yml